docs/_articles/en/user-guide/vscode-commands.md

```diff
@@ -179,6 +179,9 @@ Read on to learn about the commands available in Salesforce Extension for VS Cod
 - **SFDX: Create Aura Interface**
   - Creates an Aura interface with the specified name in the default directory `force-app/main/default/aura`.
   - Executes the CLI command `force:lightning:interface:create`
+- **SFDX: Create Lightning Web Component Test**
+  - Creates a test for a Lightning web component with the specified filepath
```

Comment:
```suggestion
  - Creates a test for a Lightning web component with the specified component name.
```
Since we prompt users for the component name instead of a file path. @ekapner @hbc-bindu how does this doc site command reference look?

Reply:
@brpowell , looks good. If the test is created for the specified component, we could shorten the message as:
Creates a test for the specified Lightning web component.